Sources/mParticle-Appboy/MPKitAppboy.m

```diff
@@ -377,6 +377,9 @@ - (void)start {
     [self->appboyInstance setInAppMessagePresenter:inAppMessageUI];
 #endif
     
+    FilteredMParticleUser *currentUser = [[self kitApi] getCurrentUserWithKit:self];
+    [self updateUser:currentUser request:currentUser.userIdentities];
```

Comment:
Should we first check if currentUser is nil before calling this?

Reply:
The `updateUser:` method accepts and handles nil users appropriately.